planet/subscription_request.py

```diff
@@ -212,6 +219,86 @@ def catalog_source(
     return {"type": "catalog", "parameters": parameters}
 
 
+def planetary_variable_source(
+    var_type: Literal["biomass_proxy",
```

Comment:
Just wanted to point out that internally we are already using more sources for upcoming forest products. 

I'm not sure if there is a way to not duplicate these lists.

Reply:
Just saw #976 after this. Moving to that MR.